helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -7,6 +7,8 @@
 nameOverride: ""
 # fullnameOverride completely replaces the generated name.
 fullnameOverride: ""
+# namespaceOverride replaces namespace in templates.
```

Comment:
```suggestion
# namespaceOverride can be used override the deployment namespace for collector resources
# Useful when including this chart as a subchart, so it can be released into a different namespace than the parent
```